tests/unit/utils.py

```diff
@@ -38,3 +42,74 @@ def dedent_plan(plan: str) -> str:
     result = dedent(plan)
     result = result.replace("    ", "\t")  # replace 4 spaces with tap
     return result
+
+
+def get_status():
```

Comment:
Question: why do we also want to read json-formatted status? Is the yaml format not enough?

Reply:
This is used in another tests and I simply moved it to utils from conftest, since it make much more sense.